app/models/media.rb

```diff
@@ -36,6 +37,8 @@ class Media < ApplicationRecord
 
   validates :item, :kind_value, presence: true
   validate :update_item_error
+  validate :non_image_cannot_be_featured
+  validate :one_featured_image_per_loan
```

Comment:
This doesn't really behave like a validation, this should probably be in some kind of a callback or something instead.